commands/Moderation/mute.js

```diff
@@ -0,0 +1,36 @@
+exports.run = async (client, msg, [user]) => {
+  const { muteRole } = msg.guild.settings;
+  if (!msg.guild.settings.muteRole) return msg.reply("Sorry you have not created a mute role for this server.");
+  const value = user.roles.get(muteRole);
```

Comment:
What if you get the role from Guild.roles, and check if the user has said role by id?